hooks/useKamonNFT.ts

```diff
@@ -0,0 +1,17 @@
+import { useNetwork, useSigner } from 'wagmi'
+import { ethers } from 'ethers'
+import { PodCastNFT__factory } from '@/contracts/types/factories/PodCastNFT__factory'
+import { getContractAddress } from 'utils/contractAddress'
```

Comment:
utils/contractAddress is written under the assumption that what is defined here will be included.
https://github.com/henkaku-center/membership-nft-interface/pull/19/files#diff-f8739af1425d55c55220f8008481e6b10d63585e5dddd23f26bf651d85b22160R1